openadapt/record.py

```diff
@@ -696,6 +731,7 @@ def read_window_events(
     terminate_processing: multiprocessing.Event,
     recording: Recording,
     started_counter: multiprocessing.Value,
+    window_event_map: dict,
```

Comment:
Please rename to something more descriptive, e.g. `window_event_by_id`

Reply:
Done in [8dabc63](https://github.com/OpenAdaptAI/OpenAdapt/pull/842/commits/8dabc634da9fe79fcee89d4bd9eed64bd81613b4)